tests/stubtest_allowlists/py36.txt

```diff
@@ -25,6 +25,7 @@ enum.Enum._generate_next_value_
 fractions.Fraction.__new__  # overload is too complicated for stubtest to resolve
 hmac.HMAC.__init__
 importlib.metadata  # Added in 3.8
+importlib.metadata._meta  # Added in 3.10
```

Comment:
I think we won't need these allowlist overrides, once mypy 0.920 gets released as it includes python/mypy#11069.